docs/index.d.ts

```diff
@@ -1021,6 +1021,44 @@ export interface RosettaNetworkStatusResponse {
   peers: RosettaPeers[];
 }
 
+export interface FTMetadataResponse {
+  /**
+   * Identifies the asset to which this token represents
+   */
+  name: string;
+  /**
+   * Describes the asset to which this token represents
+   */
+  description: string;
+  /**
+   * A URI pointing to a resource with mime type image/* representing the asset to which this token represents. The API may provide a URI to a cached resource, dependending on configuration. Otherwise, this can be the same value as the canonical image URI.
+   */
+  image_uri: string;
+  /**
+   * The original image URI specified by the contract. A URI pointing to a resource with mime type image/* representing the asset to which this token represents. Consider making any images at a width between 320 and 1080 pixels and aspect ratio between 1.91:1 and 4:5 inclusive.
+   */
+  image_canonical_uri: string;
+}
+
+export interface NFTMetadataResponse {
+  /**
+   * Identifies the asset to which this token represents
+   */
+  name: string;
+  /**
+   * Describes the asset to which this token represents
+   */
+  description: string;
+  /**
+   * A URI pointing to a resource with mime type image/* representing the asset to which this token represents. The API may provide a URI to a cached resource, dependending on configuration. Otherwise, this can be the same value as the canonical image URI.
+   */
+  image_uri: string;
+  /**
+   * The original image URI specified by the contract. A URI pointing to a resource with mime type image/* representing the asset to which this token represents. Consider making any images at a width between 320 and 1080 pixels and aspect ratio between 1.91:1 and 4:5 inclusive.
+   */
+  image_canonical_uri: string;
+}
+
```

Comment:
I'm curious, where did this schema originate? it doesn't follow the standards that have been worked on: 
https://github.com/stacksgov/sips/blob/friedger-main/sips/sip-009/sip-009-nft-standard.md
https://github.com/stacksgov/sips/blob/hstove-feat/sip-10-ft/sips/sip-010/sip-010-fungible-token-standard.md

Reply:
Which part @aulneau? The canonical image URI was my suggestion to factor in how an API instance should expose a resource like an image to clients.

Reply:
ideally for the FT schema these fields would be available:

- `symbol`
- `decimals`